core/views.py

```diff
@@ -42,6 +46,13 @@ def viewNewsletters(request):
         'NEWSLETTER_ARCHIVE_URL': global_preferences['newsletter__share_link'],
     })
 
+class AccountMixin(LoginRequiredMixin):
+    """
+        Sets the view's object to the user that makes
+        the request.
+    """
+    def get_object(self, queryset=None):
+        return self.request.user
```

Comment:
I'd argue that you can move `get_object` straight to `AccountChangeView` as it's not used anywhere else. Furthermore, `AccountChangeview` can then directly inherit `LoginRequiredMixin`